static/css/perdiem.css

```diff
@@ -437,3 +473,215 @@ li.home-logo {
     padding: 20px;
     max-width: 600px;
 }
+
+#percentage {
+    color: #4cbd7e;
+}
+
+#percentage1 {
+    color: #4cbd7e;
+    font-size: 18px;
+}
+
+.profile-invested {
+    font-size: 20px;
+    color: #4d96ad;
+}
+
+.profile-earned {
+    font-size: 20px;
+    color: #4d96ad;
+}
+
+.roi {
+    font-size: 20px;
+    color: #4d96ad;
+    text-transform: uppercase;
+    font-family: Dosis;
+    font-weight: 400;
+}
+
+ .tabs-title>a {
+    padding: 5px;
+    height: 100%;
+    position: vertical-align;
+ }
+
+ .tab-icon {
+    padding: 5px;
+    color: #4d96ad;
+ }
+
+ .menu>li>a {
+    padding-right: 15px;
+    padding-left: 15px;
+    padding-top: 7px;
+    padding-bottom: 7px;
+ }
+
+ a.nav-icon {
+    padding: 0px;
+ }
+
+ li.home-logo {
+    padding: 2px;
+ }
+
+.name {
+    font-family: Dosis;
+    font-size: 20px;
+    text-transform: uppercase;
+    color: #4d96ad;
+}
+
+.profile-container {
+  width: 80%;
+  margin: 15px auto;
+}
+
+#myChart {
+    font-family: Dosis;
+}
+
+h2.nav-name {
+    text-align: center;
+    font-size: 18px;
+    padding: 5px;
+    margin: 0px;
+}
+
+.profile-stats {
+    margin: 0 auto;
+}
+
+.login-item {
+    padding: 10px;
+}
+
+.left-align {
+    text-align: left
+}
+
+.signup-manual {
+    background-color: #85929E;
+    color: #fff;
+    padding-left: 60px;
+    padding-right: 60px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    border-radius: 5px;
+    font-weight: 400 !important;
+}
+
+#register-page {
+    position: relative;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-color: black;
+    height: 80vh;
+    }
+
+.signup-form {
+    margin-top: 10%;
+}
+
+.register-copy {
+    color: #285e6f;
+    padding: 20px;
+    font-family: ;
+}
+
+.privacy-padding {
+    padding: 40px;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+.terms-padding {
+    padding: 40px;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+.faq-padding {
+    padding: 40px;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+.contact-padding {
+    padding: 40px;
+    max-width: 700px;
+    margin: 0 auto;
+}
+
+.top-bar-left {
+    width: auto !important;
```

Comment:
Okay did a pretty good amount of updates to the CSS. Still couldn't figure out how to override a couple that are bing pulled from foundation, but I'll go back in a bit and take another shot. Let me know if there is anything that you see which is obvious that I'm missing.